website/content/docs/installation/source.mdx

```diff
@@ -2,78 +2,82 @@
 layout: docs
 page_title: Installing Vagrant from Source
 description: |-
-  Installing Vagrant from source is an advanced topic and is only recommended
-  when using the official installer is not an option. This page details the
-  steps and prerequisites for installing Vagrant from source.
+  Vagrant installations from source is an advanced operation. It is only recommended
+  if you cannot use the official installer.
 ---
 
-# Installing Vagrant from Source
+# Install Vagrant from source
 
-Installing Vagrant from source is an advanced topic and is only recommended
-when using the official installer is not an option. This page details the
-steps and prerequisites for installing Vagrant from source.
+Vagrant installations from source is an advanced operation. It is only recommended
+if you cannot use the official installer. This page details the prerequisites and three
+steps to install Vagrant from source.
+
+1. Install Ruby
+1. Clone Vagrant
+1. Configure locally
 
 ## Install Ruby
 
-You must have a modern Ruby (>= 2.2) in order to develop and build Vagrant. The
-specific Ruby version is documented in the Vagrant's `gemspec`. Please refer to
-the `vagrant.gemspec` in the repository on GitHub, as it will contain the most
-up-to-date requirement. This guide will not discuss how to install and manage Ruby.
-However, beware of the following pitfalls:
+You must install Ruby 2.7 or newer to develop and build Vagrant. The
+specific Ruby version that you will need is documented 
+in `vagrant.gemspec`, located in the repository on GitHub. 
+It contains the most up-to-date requirements. 
+
+~> **Note:** It is not recommended that you use the system Ruby. You need a Ruby version manager like RVM or chruby.
```

Comment:
```suggestion
~> **Note:** We do not recommended that you use the system Ruby. You need a Ruby version manager like RVM or chruby.
```

Reply:
This note can be discarded now. This is less of an issue anymore now that recent rubies are generally available via system packages.